images/macos/toolsets/toolset-10.15.json

```diff
@@ -11,7 +11,7 @@
             "6.12.0.93", "6.10.0.106", "6.8.0.123", "6.6.0.166", "6.4.0.208"
         ],
         "ios-versions": [
-            "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
+            "14.2.0.12", "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
```

Comment:
New Xamarin.iOS version also means we need a new bundle with it

```
{
   "symlink": "6_12_2",
   "mono":"6.12",
   "ios": "14.2",
   "mac": "6.20",
   "android": "11.0"
}
```

Reply:
Done.